reps/2024-06-13-export-api.md

```diff
@@ -0,0 +1,141 @@
+## Summary
+### General Motivation
+
+In the current design of Ray, the way to export various states in the Ray cluster are inconsistent.
```

Comment:
I believe the State API is typically recommended for end users to fetch current state info of a running Ray cluster, but the state API gets data from multiple sources depending on the resource type. 

The export API will provide another way to fetch state data that works well for large scale clusters or can be saved to query after the cluster is terminated, but I don’t see disjoint interfaces for state data as the main motivation for this feature.

Reply:
Make sense. I mentioned it in the Key requirements: 
To obtain the current status of the ray cluster at one time, you need to use state api.